pages/services/marathon-lb/marathon-lb-advanced-tutorial/index.md

```diff
@@ -235,33 +235,34 @@ To demonstrate the `vhost` feature:
 
 1.  Add the `HAPROXY_0_VHOST` label to your local `nginx-external.json` file. In this example, the public DNS name is `brenden-j-publicsl-1ltlkzeh6b2g6-1145355943.us-west-2.elb.amazonaws.com`.
 
-        ```json
-        ...
-          "labels":{
-            "HAPROXY_GROUP":"external",
-            "HAPROXY_0_VHOST":"brenden-j-publicsl-1ltlkzeh6b2g6-1145355943.us-west-2.elb.amazonaws.com"
-          }
-        }
-        ```
+  ```json
+  ...
+    "labels":{
+      "HAPROXY_GROUP":"external",
+      "HAPROXY_0_VHOST":"brenden-j-publicsl-1ltlkzeh6b2g6-1145355943.us-west-2.elb.amazonaws.com"
+    }
+  }
+  ```
+  
   **Important:** Do not include the leading `http://`trailing slash (`/`) in your public DNS name.
 
```

Comment:
My only suggestion here is to make sure the "Important" note be formatted according to our standard. You can find the code [here](https://wiki.mesosphere.com/display/DOCS/Documentation+Style+Guide#DocumentationStyleGuide-Notes). It's not a deal-breaker, though. If you need help, I can walk you through it.